### PR TITLE
Create promoting integrations

### PR DESCRIPTION
**Pull request by tobsowo**

### CODE REVIEW

**Review by RCheesley — CHANGES_REQUESTED** [review]
Some small suggestions. As a thought you could run a poll on the community portal once a quarter to decide which ones get ranked on the homepage?

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (tobsowo) [reply pages/06.policies/09.Promoting-integrations/docs.md:21]: The three were picked at random.
The point on quaterly poll is a good suggestion and we can do that.

Comment (usmannasir) [reply pages/06.policies/09.Promoting-integrations/docs.md:21]: @RCheesley I can speak for CyberPanel. We were one of the first control panels who integrated Mautic one-click installation. 

Not only that we promoted and brought many users to Mautic. Then we've created lots of study material which have been seen by many people

https://www.youtube.com/watch?v=zeUfl0G6uOo&pp=ygURbWF1dGljIGN5YmVycGFuZWw%3D
https://www.youtube.com/watch?v=ML6NAE2h0xY&pp=ygURbWF1dGljIGN5YmVycGFuZWw%3D

Comment (RCheesley) [reply pages/06.policies/09.Promoting-integrations/docs.md:21]:
```suggestion
Projects featured in the Mautic Integrations Directory must be proposed by a member of Mautic via the [Community Portal](https://community.mautic.org/assemblies/marketing-team/f/53/proposals/new). Integrations require a minimum of five votes from the community to be considered for inclusion, and be endorsed by the Marketing Team. 

The projects listed on the Mautic website home page are selected based on their popularity and relevance to Mautic users, which is accomplished with a quarterly vote on the Community Portal of all eligible integrations. Members of Mautic are eligible to vote. 
```

@tobsowo I mean this should be documented. I believe we should use the community voting option. This is less open to personal biases and gives the community the opportunity to be engaged in the process and therefore it be more representative of sentiment. I'd suggest we limit it to members of Mautic only who are allowed to both propose integrations and vote. If quarterly proves to be too frequent we could make it 6-monthly.

I can help setting up the voting process and documenting it for you so that it's easy for you to reproduce every 3 months.

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:25]:
```suggestion
- Projects that have gained popularity within the Mautic community are more likely to be featured on the home page, as members are more likely to vote for them.
```

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:27]:
```suggestion
```

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:28]:
```suggestion
```

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [reply pages/06.policies/09.Promoting-integrations/docs.md:27]: This is far too subjective and open to bias. I think we should instead allow the community to direct this.

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:33]:
```suggestion
- The Marketing Team will assess how well a project addresses the needs of Mautic users and enhances their experience.
```

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:41]:
```suggestion
The maintenance of projects listed on the Mautic website home page is the responsibility of the Marketing Team. This ensures consistency, accuracy, and a seamless experience for visitors.
```

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:45]:
```suggestion
- The Marketing Team will regularly review and update the information related to listed projects to reflect their current status, features, and any relevant changes.
```

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:53]:
```suggestion
- The Marketing Team will communicate with project maintainers before removal, providing an opportunity for clarification or updates.
```

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:63]:
```suggestion
- The Marketing Team encourages the community to actively contribute to Open Source projects.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
The more I've thought over this, the more I feel it's too open to bias and personal views by saying the marketing team will decide. We have the tools to do this ethically and democratically, so let's use them.

I think we need a process for applying to be included in the directory in the first place (I've set up a [proposals component](https://community.mautic.org/assemblies/marketing-team/f/53/proposals/new) on the Marketing Team assembly for this) and also we could establish a regular (maybe 3 or 6 monthly) opportunity for people to decide which integrations are their favourites to go on the homepage.

I've set it that five positive votes are required for a proposed integration to be included, plus endorsement by the marketing team which @tobsowo and @sdoering can do.

I think it's sensible to use the portal for this, both because we have the technology to do this and because it means we can encourage more people/orgs to become members so they can propose projects and vote.

See what you think about the proposed changes.

**Comment by tobsowo** [comment]
I agree with the members voting and suggesting what integration is listed.

**Comment by RCheesley** [comment]
@tobsowo I am not seeing the changes implemented in the file - it looks like the conversations are marked as resolved but no changes have been made? Do you need me to make the changes that I'd suggested?

**Comment by tobsowo** [comment]
My bad. I thought the changes is done already and needed my approval.

Please, help with the changes.

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:63]: @tobsowo what do you mean by 'Partner project's here? I'm a bit concerned it might be confused with our 'Mautic Partners'. Could you maybe explain what you mean so we can word it better? Thanks!

**Review by RCheesley — CHANGES_REQUESTED** [review]
I've committed all the changes but just wanted to clarify what we mean by partner projects in the last section before merging.

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (tobsowo) [reply pages/06.policies/09.Promoting-integrations/docs.md:63]: Partners has to do with the official community partners

Comment (RCheesley) [reply pages/06.policies/09.Promoting-integrations/docs.md:63]: Ah awesome - I'll make sure that's worded clearer then!

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:55]:
```suggestion
Projects with an open source nature are given priority for listing on the Mautic website home page. Open source aligns with the collaborative and community-driven spirit of Mautic. However, projects from official [Mautic Partners](https://mau.tc/partners) may also be considered based on their significance and value to the Mautic ecosystem.
```

File: pages/06.policies/09.Promoting-integrations/docs.md
Comment (RCheesley) [inline pages/06.policies/09.Promoting-integrations/docs.md:65]:
```suggestion
- Projects from official Mautic Partners may be considered, even if not open source, if they provide substantial value to the Mautic community.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
See what you think about these small tweaks to make it clearer about official partner projects!

**Comment by RCheesley** [comment]
OK let's get this merged! 🚀

**Pull request merged**
